Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -916,7 +920,9 @@ extension BrowserViewController: HomePanelViewControllerDelegate {
         finishEditingAndSubmit(url, visitType: visitType)
     }
     func homePanelViewController(homePanelViewController: HomePanelViewController, didSelectPanel panel: Int) {
-        tabManager.selectedTab?.lastHomePanel = panel
+        if AboutUtils.isAboutHomeURL(tabManager.selectedTab!.url!) {
+            tabManager.selectedTab?.webView?.evaluateJavaScript("history.replaceState({}, '', '/about/home/#panel=\(panel)')", completionHandler: nil)
```

Comment:
Can you try testing with just `#panel=X` instead of `/about/home/panel=X`? I'm curious whether we even need to include the path here.